Gemfile

```diff
@@ -5,5 +5,6 @@ gem "insist"
 gem "stud"
 gem "fpm"
 gem "pleaserun"
+gem "jls-lumberjack", :github => "elastic/ruby-lumberjack"
```

Comment:
I decided to point to the github repo so we can setup jenkins to trigger a LSF build after a lumberjack change? Using `gem "jls-lumberjack"` will only test against the stable release.